package.json

```diff
@@ -61,8 +61,10 @@
   "dependencies": {
     "@babel/preset-react": "^7.16.7",
     "@bigbinary/neeto-icons": "^1.8.17",
-    "@bigbinary/neetoui": "^3.2.29",
+    "@bigbinary/neetoui": "^4.0.3",
+    "@emotion/is-prop-valid": "^1.2.0",
     "@rails/webpacker": "5.4.3",
+    "antd": "^4.23.1",
```

Comment:
Just to confirm. Does neetoUI v4 require this package as whole?

Reply:
From compilation error I checked we are using antd/lib/date-picker and antd/lib/table but how to install them specifically?